### ISSUE TITLE
Install fails via pip

### ISSUE THREAD

**Issue by mattwilson1024** [thisbejim/Pyrebase#38]
I get the following error trying to `pip install pyrebase`.

I'm running OSX and Python 3.5.1. New to python so I'm pretty stumped!

```
Collecting grpcio
  Using cached grpcio-0.14.0.tar.gz
Requirement already satisfied (use --upgrade to upgrade): six>=1.10 in /Users/matt/.virtualenvs/cv3/lib/python3.5/site-packages (from grpcio)
Requirement already satisfied (use --upgrade to upgrade): enum34>=1.0.4 in /Users/matt/.virtualenvs/cv3/lib/python3.5/site-packages (from grpcio)
Requirement already satisfied (use --upgrade to upgrade): futures>=2.2.0 in /Users/matt/.virtualenvs/cv3/lib/python3.5/site-packages (from grpcio)
Requirement already satisfied (use --upgrade to upgrade): protobuf>=3.0.0a3 in /Users/matt/.virtualenvs/cv3/lib/python3.5/site-packages (from grpcio)
Requirement already satisfied (use --upgrade to upgrade): setuptools in /Users/matt/.virtualenvs/cv3/lib/python3.5/site-packages (from protobuf>=3.0.0a3->grpcio)
Building wheels for collected packages: grpcio
  Running setup.py bdist_wheel for grpcio ... error
  Complete output from command /Users/matt/.virtualenvs/cv3/bin/python3.5 -u -c "import setuptools, tokenize;__file__='/private/var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-build-v2v0619p/grpcio/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" bdist_wheel -d /var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/tmpw80x2gappip-wheel- --python-tag cp35:
  /Users/matt/.virtualenvs/cv3/lib/python3.5/site-packages/setuptools/command/easy_install.py:333: UserWarning: Unbuilt egg for grpcio [unknown version] (/private/var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-build-v2v0619p/grpcio/src/python/grpcio)
    self.local_index = Environment(self.shadow_path + sys.path)
  running bdist_wheel
  running build
  running build_py
  running build_proto_modules
  Traceback (most recent call last):
    File "/private/var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-build-v2v0619p/grpcio/src/python/grpcio/commands.py", line 227, in run
      self.run_command('build_proto_modules')
    File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/dist.py", line 974, in run_command
      cmd_obj.run()
    File "/private/var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-build-v2v0619p/grpcio/src/python/grpcio/commands.py", line 159, in run
      raise CommandError('could not find protoc')
  commands.CommandError: could not find protoc
  
  During handling of the above exception, another exception occurred:
  
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/private/var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-build-v2v0619p/grpcio/setup.py", line 246, in <module>
      test_runner=TEST_RUNNER,
    File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/core.py", line 148, in setup
      dist.run_commands()
    File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/dist.py", line 955, in run_commands
      self.run_command(cmd)
    File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/dist.py", line 974, in run_command
      cmd_obj.run()
    File "/Users/matt/.virtualenvs/cv3/lib/python3.5/site-packages/wheel/bdist_wheel.py", line 179, in run
      self.run_command('build')
    File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/dist.py", line 974, in run_command
      cmd_obj.run()
    File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/command/build.py", line 135, in run
      self.run_command(cmd_name)
    File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/cmd.py", line 313, in run_command
      self.distribution.run_command(command)
    File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/dist.py", line 974, in run_command
      cmd_obj.run()
    File "/private/var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-build-v2v0619p/grpcio/src/python/grpcio/commands.py", line 229, in run
      sys.stderr.write('warning: %s\n' % error.message)
  AttributeError: 'CommandError' object has no attribute 'message'
  
  ----------------------------------------
  Failed building wheel for grpcio
  Running setup.py clean for grpcio
Failed to build grpcio
Installing collected packages: grpcio
  Running setup.py install for grpcio ... error
    Complete output from command /Users/matt/.virtualenvs/cv3/bin/python3.5 -u -c "import setuptools, tokenize;__file__='/private/var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-build-v2v0619p/grpcio/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-0zyt3z6f-record/install-record.txt --single-version-externally-managed --compile --install-headers /Users/matt/.virtualenvs/cv3/bin/../include/site/python3.5/grpcio:
    /Users/matt/.virtualenvs/cv3/lib/python3.5/site-packages/setuptools/command/easy_install.py:333: UserWarning: Unbuilt egg for grpcio [unknown version] (/private/var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-build-v2v0619p/grpcio/src/python/grpcio)
      self.local_index = Environment(self.shadow_path + sys.path)
    running install
    running build
    running build_py
    running build_proto_modules
    Traceback (most recent call last):
      File "/private/var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-build-v2v0619p/grpcio/src/python/grpcio/commands.py", line 227, in run
        self.run_command('build_proto_modules')
      File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/dist.py", line 974, in run_command
        cmd_obj.run()
      File "/private/var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-build-v2v0619p/grpcio/src/python/grpcio/commands.py", line 159, in run
        raise CommandError('could not find protoc')
    commands.CommandError: could not find protoc
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-build-v2v0619p/grpcio/setup.py", line 246, in <module>
        test_runner=TEST_RUNNER,
      File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/core.py", line 148, in setup
        dist.run_commands()
      File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/dist.py", line 955, in run_commands
        self.run_command(cmd)
      File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/dist.py", line 974, in run_command
        cmd_obj.run()
      File "/Users/matt/.virtualenvs/cv3/lib/python3.5/site-packages/setuptools/command/install.py", line 61, in run
        return orig.install.run(self)
      File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/command/install.py", line 539, in run
        self.run_command('build')
      File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/dist.py", line 974, in run_command
        cmd_obj.run()
      File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/command/build.py", line 135, in run
        self.run_command(cmd_name)
      File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/distutils/dist.py", line 974, in run_command
        cmd_obj.run()
      File "/private/var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-build-v2v0619p/grpcio/src/python/grpcio/commands.py", line 229, in run
        sys.stderr.write('warning: %s\n' % error.message)
    AttributeError: 'CommandError' object has no attribute 'message'
    
    ----------------------------------------
Command "/Users/matt/.virtualenvs/cv3/bin/python3.5 -u -c "import setuptools, tokenize;__file__='/private/var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-build-v2v0619p/grpcio/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-0zyt3z6f-record/install-record.txt --single-version-externally-managed --compile --install-headers /Users/matt/.virtualenvs/cv3/bin/../include/site/python3.5/grpcio" failed with error code 1 in /private/var/folders/cm/jhy671nj51bf80148gr7grwr0000gn/T/pip-build-v2v0619p/grpcio/
```

**Comment by thisbejim** [comment]
Hi @mattwilson1024, thanks for writing in, I just checked this out and am getting this error too. 

This seems to be an issue with the [grpcio](https://github.com/grpc/grpc) package, you can follow the progress [here](https://github.com/GoogleCloudPlatform/gcloud-python/issues/1879) and [here](https://github.com/grpc/grpc/issues/6939), but I will also update this thread when the issue is resolved.

**Comment by thisbejim** [comment]
Fixed with GoogleCloudPlatform/gcloud-python#1938 :smiley:
